### PR TITLE
Update @classmethod.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: commands/__init__.py [hunk 1]
```diff
@@ -6,6 +6,8 @@ class CommandError(Exception):
 
 
 class BotCommand(object):
+    command = ''
```
Comment (glogiotatidis) [inline commands/__init__.py]: `None` is nicer imho.

File: commands/__init__.py [hunk 2]
```diff
@@ -25,6 +27,8 @@ def queue(self, message, next_cmd):
 
     @classmethod
     def match(cls, message):
```
Comment (glogiotatidis) [inline commands/__init__.py:29]: Just delete `match` altogether, no need to overload it here or in other commands.

**Comment by glogiotatidis** [comment]
Tests fail.

File: commands/__init__.py (hunk 1 above)
Comment (akatsoulas) [inline commands/__init__.py]: Personally, I prefer an empty string for strings but I 'll go with None :)

**Comment by akatsoulas** [comment]
pr updated and travis is happy :)

**Comment by akatsoulas** [comment]
@glogiotatidis r?

**Comment by glogiotatidis** [comment]
ωπ το ξέχασα αύτο :cake:

**Pull request merged**